susemanager-utils/testing/automation/java-checkstyle.sh

```diff
@@ -4,5 +4,42 @@ HERE=`dirname $0`
 . $HERE/VERSION
 GITROOT=`readlink -f $HERE/../../../`
 
-docker pull $REGISTRY/$PGSQL_CONTAINER
-docker run --privileged --rm=true -v "$GITROOT:/manager" $REGISTRY/$PGSQL_CONTAINER /manager/java/scripts/docker-checkstyle.sh
+help() {
+  echo ""
+  echo "Script to run a docker container to verify java code style"
+  echo ""
+  echo "Syntax: "
+  echo ""
+  echo "${SCRIPT} -c /path/to/oscrc"
```

Comment:
Same here.

Reply:
defaulting to `${HOME}/.osrc`: this avoids the need of any change in the CI